core-bundle/tests/Framework/ContaoFrameworkTest.php

```diff
@@ -277,6 +260,10 @@ public function testOverridesTheErrorLevel(): void
         error_reporting($errorReporting);
     }
 
+    /**
+     * @runInSeparateProcess
```

Comment:
Why did you re-add this? The tests in the master branch work without a separate process.